frontend/pages/sacfairguide.tsx

```diff
@@ -0,0 +1,285 @@
+      alt={'Manage Club Button'}
+      title={'Manage Club Button'}
+    />
+    <p>Then, click the "Events" tab:</p>
+    <img
+      style={{
+        maxHeight: 400,
+        verticalAlign: 'middle',
+        margin: 10,
+        padding: 10,
+      }}
+      src={'static/img/sacfair_screencaps/2.png'}
+      alt={'Events Tab'}
+      title={'Events Tab'}
+    />
+
+    <p>
+      There will be an existing event with your designated date & time titled
+      "SAC Fair Info Session". Click "Edit".
+    </p>
+    <img
+      style={{
+        maxHeight: 400,
+        verticalAlign: 'middle',
+        margin: 10,
+        padding: 10,
+      }}
+      src={'static/img/sacfair_screencaps/3.png'}
+      alt={'Edit SAC Fair Event'}
+      title={'Edit SAC Fair Event'}
+    />
+    <p>
+      Edit the club details: you <strong>MUST</strong> provide a Zoom link
+      (preferably from your school Zoom account) where prospective students can
+      navigate to. You can add a description, as well as upload a cover photo
+      (dimensions 16:9) to make your event stand out in the live events portal.
+      Be sure to{' '}
+      <strong>
+        hit the green "save" button on the lefthand side when you are done!
+      </strong>
+    </p>
+
+    <img
+      style={{
+        maxHeight: 400,
+        verticalAlign: 'middle',
+        margin: 10,
+        padding: 10,
+      }}
+      src={'static/img/sacfair_screencaps/4.png'}
+      alt={'Save SAC Fair Changes'}
+      title={'Save SAC Fair Changes'}
+    />
+
+    <Subheading>Zoom Best Practices</Subheading>
+    <p>
+      To get all possible features, please use an account using the school's
+      licensed Zoom accounts. You should have received an email from your home
+      school about obtaining login credentials for this. Schedule a new meeting,
+      using the <strong>given date and time</strong> as outlined in the e-mail
+      given to you by SAC with these settings:
+    </p>
+    <UnorderedList>
+      <li>A required meeting password</li>
+      <li>Under Advanced Options:</li>
+      <UnorderedList>
+        <li>
+          Waiting room – <strong>disabled</strong>
+        </li>
+        <li>
+          Mute participants upon entry – <strong>enabled</strong>
+        </li>
+        <li>
+          Only authenticated users can join – <strong>enabled</strong>
+        </li>
+      </UnorderedList>
+    </UnorderedList>
+
+    <p>When hosting your event:</p>
+    <UnorderedList>
+      <li>
+        Make all present mebers of the session <strong>co-hosts</strong> to help
+        control the session.
+      </li>
+      <li>
+        Have the <strong>chat window open</strong> to answer any questions. For
+        busier sessions, we recommend designating one member to be monitoring
+        the chat window
+      </li>
+      <li>
+        For busier sessions where multiple club members are present, we
+        recommend taking advantage of the <strong>breakout room feature</strong>
```

Comment:
We should warn users that they need to set up breakout rooms *beforehand* and enable the settings on their account, so they don't try to use it right before the meeting starts and have to redo their Zoom links.

Reply:
Also would be great to add screenshots of exactly what settings to enable, I know this has been a pain point in the past.